projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/agents-button/agents-button.component.scss

```diff
@@ -45,7 +45,14 @@ aca-agents-button.aca-agents-button {
 
       &-content {
         display: flex;
-        align-items: baseline;
+        align-items: center;
+
+        &-name {
+          width: 120px;
+          white-space: nowrap;
+          overflow: hidden;
+          text-overflow: ellipsis;
```

Comment:
I see you added ellipsis in case text is too long. It's fine. But can you also add some tooltips? Actually when name is cut then user can't see full name so tooltip would be helpful: 
<img width="534" alt="image" src="https://github.com/user-attachments/assets/f1162cd6-9af6-44c8-a8aa-ae78550df9e2">